script/Ciel/Day 1/01_01_CIEL01_2_3.txt

```diff
@@ -752,7 +752,7 @@ Didn't you see the bulletin board at school? Students were asked to volunteer."
 -- Page 114, Offset 767.
 -- でも、そういうコトするから教師連中がアテにするんスよ。先公どもは隙あらばサボりやがるんだから、あんまり調子に乗らせないでほしいんですけどねぇ」
 The teachers rely on you because you do stuff like that though.
- They'll slack off any chance they get, so don't get carried away and allow them to just coast."
+ They'll slack off any chance they get, so I kinda wish you didn't let them get away with that."
```

Comment:
"kinda wish you wouldn't let them get away with it"

Reply:
They can't keep getting away with ittttttt